sketch.js

```diff
@@ -1,137 +1,161 @@
-let shapeClassifier;
-let canvas;
-let img;
-let buttons;
-
-let labels = [
-  "square",
-  "circle",
-  "triangle",
-]
-let label = labels[0];
-let drawing_array = [];
-let drawing = [];
-let i = 0;
+// GLOBALS
+let shapeClassifier;  // ml5.neuralnetwork()
+let canvas;  // createCanvas()
+
+// LABELS AND CYCLING
+let labels = ["square","circle","triangle"];  
+let i = 0;  // MOVE INTO LOCAL
+let current_label = labels[0];  // defaults drawing label to square
+
+// DRAWINGS
+let DRAW = {
+  FREE:[],
+  LINE:{
+    x1:undefined,
+    y1:undefined,
+    x2:undefined,
+    y2:undefined,
+  },
+  free_store:[],
+  line_store:[]
+}
```

Comment:
Capitalized variable names should be treated as private, i.e. not to be explictly modified. One should use the variables named with lowercase whenever one wants to add or remove shapes.